metron-analytics/metron-profiler-storm/src/test/java/org/apache/metron/profiler/storm/integration/ProfilerIntegrationTest.java

```diff
@@ -503,21 +496,29 @@ public void setup() {
       put(PROFILER_SALT_DIVISOR.getKey(), saltDivisor);
     }};
 
+    Context context = new Context.Builder()
+            .with(Context.Capabilities.GLOBAL_CONFIG, () -> global)
+            .build();
+
+    // create the GET_PROFILE function
+    GetProfile getProfileFunction = new GetProfile()
+            .withProfilerClientFactory(new HBaseProfilerClientFactory(new FakeHBaseClientFactory()));
```

Comment:
We need both the Stellar functions and the topology running in Storm to "see" the same "HBase data" even though we can't run HBase with this integration test.  Both the topology and the functions see the same static data presented by the `FakeHBaseClient`.

Reply:
Just to clarify - is the `FakeHBaseClient` the replacement for our mock hbase table impl or something different entirely?

Reply:
Different tests needed different replacements. But in many cases, especially the integration tests, using a `FakeHBaseClient` is the replacement for the "mock Hbase table stuff".

Reply:
Just so I'm clear, by the end of the upgrade we will no longer have the existing mock hbase, just the new fake client?

Reply:
Correct.  You will see that change in #1470 .